src/libs/ReportActionsUtils.ts

```diff
@@ -435,6 +439,18 @@ function isConsecutiveActionMadeByPreviousActor(reportActions: ReportAction[] |
         return false;
     }
 
+    if (isReportSubmitAction(currentAction)) {
+        const currentActionAdminAccountID = currentAction.adminAccountID;
+
+        return currentActionAdminAccountID === previousAction.actorAccountID || currentActionAdminAccountID === previousAction.adminAccountID;
+    }
+
+    if (isReportSubmitAction(previousAction)) {
+        return typeof previousAction.adminAccountID === 'number'
```

Comment:
what is the puropse of returning `typeof`? Is it a typo?

Reply:
The idea is to check if `adminAccountID` exists actually in a report action. It is a `number` or `undefined`. So, it is important to verify that we can lean on the value.